src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -708,6 +708,9 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 }
             }
 
+            // All nodes that have runtime warnings or infos
+            HashSet<Guid> nodesWithInfos = [.. warnings.Keys];
```

Comment:
Can you explain this change? It looks like the guids need to be unique? What if a node has both a warning and info message - won't there be repeated guids in that case?

Reply:
these node guids are used to decide if we need to update the bubble info model and UI after a node has been executed. 
The guid represents nodes (not warnings/infos)
https://github.com/DynamoDS/Dynamo/blob/7a58fa79a95bd7850b114c4d6cb51218a2b6975e/src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs#L355-L366